x/incentives/keeper/distribute_test.go

```diff
@@ -1131,3 +1132,223 @@ func (s *KeeperTestSuite) IncentivizeInternalGauge(poolIds []uint64, epochDurati
 	)
 	s.Require().NoError(err)
 }
+func (s *KeeperTestSuite) TestAllocateAcrossGauges() {
```

Comment:
Is there a particular reason we don't have table driven tests for this logic?

Reply:
i can make it table driven, there was no functional params to test on so i lazily added regular test.

however I can test test output on certain scenarios

Reply:
convert to table driven: https://github.com/osmosis-labs/osmosis/pull/6085/commits/c99cb1de19308284b9f84184660245572de9bdda#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR1128